Remotion/Web/UnitTests/Core/UI/Controls/WebButtonTests/WebButtonDiagnosticMetadataTest.cs

```diff
@@ -45,6 +45,20 @@ public void RenderDiagnosticMetadataAttributes ()
       Assert.That (renderedText, Does.Contain (DiagnosticMetadataAttributes.TriggersPostBack + "=\"true\""));
     }
 
+    [Test]
+    public void RenderDiagnosticMetadataAttributes_WithHtmlText_StripsTagsInContent ()
+    {
+      var webButton = new TestWebButton { ID = "WebButton", Text = WebString.CreateFromHtml ("<p>My Button</p>") };
+
+      var renderedText = RenderControl (webButton);
+
+      Assert.That (renderedText, Does.Contain (DiagnosticMetadataAttributes.ControlType + "=\"WebButton\""));
+      Assert.That (renderedText, Does.Contain (DiagnosticMetadataAttributes.ItemID + "=\"" + webButton.ID + "\""));
+      Assert.That (renderedText, Does.Contain (DiagnosticMetadataAttributes.Content + "=\"My Button\""));
+      Assert.That (renderedText, Does.Not.Contains (DiagnosticMetadataAttributes.CommandName));
+      Assert.That (renderedText, Does.Contain (DiagnosticMetadataAttributes.TriggersPostBack + "=\"true\""));
```

Comment:
Please drop the redundant assertions (trigger, controltype, etc) that are not part of the current test's focus.